docs/reference-implementation/Getting-Started.md

```diff
@@ -28,7 +28,7 @@ This guide offers two paths: Using Azure DevOps Portal or script-based via Azure
 The process to deploy AlwaysOn is comprised of the following steps:
 
 1) Create an [Azure DevOps organization and project](#create-a-new-azure-devops-project)
-1) Generate your own repository based on the [AlwaysOn GitHub template](https://github.com/Azure/AlwaysOn-Foundational-Online/generate) repository
+1) Generate your own repository based on the [AlwaysOn GitHub template](https://github.com/Azure/Mission-Critical-Foundational-Online/generate) repository
```

Comment:
should just be "Mission-Critical-online"

Reply:
fixed